packages/corbado_auth/README.md

```diff
@@ -171,8 +172,14 @@ You can also close and reopen the app.
 You should still be logged in.
 
 <p float="left">
-    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/signup.jpeg" height="500" alt="signup">
-    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/signup_fingerprint.jpeg" height="500" alt="signup fingerpring">
-    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/home.jpeg" height="500" alt="home">
-    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/idtoken.jpeg" height="500" alt="id token">
+    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/signin-signup-image.jpg" height="500" alt="signup">
+    <img src="https://raw.githubusercontent.com/corbado/flutter-passkeys/main/packages/corbado_auth/doc/img/profile-token-details-image.jpg" height="500" alt="signup fingerpring">
 </p>
+
+## Troubleshooting
+
+| Type                            | Issue                     | Note                                                                                                                             |
+|---------------------------------|---------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| iOS Sim                         | Localizd string not found | For passkeys to work in the iOS simulator, you need to enable faceID first in the top menu bar under features > faceID > enrolled |
+| iOS physical device             | Passkeys not working      | Passkeys do not work when running your flutter application on your physical iOS device. This has to do with the signing of your app. Rather test on a virtual iOS device or virtual/ physical Android device.                                                                                                                                 |
```

Comment:
This comment is about the example right? (for general applications it must of course be possible to run on a physical device)

Reply:
yes. as far as I remember you told me about this because there is some issue regarding associated domains